File: app/src/main/java/com/example/webnovellibrary/Folder.java
package com.example.webnovellibrary;

import java.util.ArrayList;
import java.util.List;

public class Folder {
    String name;
    List<WebNovel> webNovels;

    public Folder(String folderName) {
        this.name = folderName;
        this.webNovels = new ArrayList<>();
    }
    public String getFolderName() {
        return name;
    }

    public void setFolderName(String folderName) {
        this.name = folderName;
    }

    public List<WebNovel> getWebNovels() {
        return webNovels;
    }

    public void setWebNovels(List<WebNovel> webNovels) {
        this.webNovels = webNovels;
    }

}

File: app/src/main/java/com/example/webnovellibrary/Folder.kt
package com.example.webnovellibrary

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Folder (
    var name: String,
    var webNovels: MutableList<WebNovel> = mutableListOf<WebNovel>()
    ): Parcelable